packages/server/src/cli.ts

```diff
@@ -37,6 +37,7 @@ commander.version(process.env.VERSION || "development")
 Error.stackTraceLimit = Infinity;
 if (isCli) {
 	require("nbin").shimNativeFs(buildDir);
+	require("nbin").shimNativeFs("/node_modules");
```

Comment:
I'm not 100% on whether this is a safe thing to do. Maybe we should just replace `bindings` to give the path to the `.node` file instead? `bindings` is what is trying to read the `/node_modules` path to find the `.node` file, so we could bypass it and give it directly.